### DISCUSSION TITLE
Docker image configuration

### DISCUSSION THREAD

**Discussion by kedith** [datreeio/datree#698] [Q&A] [answered]
Hi,

I would like to use the _datree/datree_  image and configure it for offline use with no internet connection. Additionally, I want to extend the image and add kubernetes schema files for validation and a policies.yaml. I tried to look inside the docker container but I can't seem to find the usual config files which are created when running the executable normally. Where are the default configs located in the docker image? 

Thanks,
Edith

**Comment by adifayer** [comment, marked as answer]
Hey @kedith 
Currently, running datree in offline-mode within our docker image is not supported out-of-the-box. I've opened a task to support this use case- [https://github.com/datreeio/datree/issues/700](url)

As a workaround, you can create your own image using our image as a base, and extend it to perform the following:
1. Execute: `datree config set offline local` which creates the config.yaml file with the property that enables running offline
2. Datree test 

Let me know if it solved your problem,
Adi.

**Reply by kedith** [reply]
Hey @adifay,

Thanks, I tried it out and it works: I created my own image from the datree base image where I run that command.

Have a nice day,
Edith

**Reply by adifayer** [reply]
Happy to hear! 

Have a wonderful day,
Adi
